greek/use_the_rope/use_the_rope.md

```diff
@@ -2,10 +2,14 @@
 
 Κάνεις ένα κόμπο και δοκιμάζεις να δεις ότι είναι σταθερά δεμένο πάνω  σου
 
-Τραβάς την το υπόλοιπο για να βρεις την άλλη άκρη και διαπιστώνεις ότι το υπόλοιπο σχοινί έρχεται μέσα από το ζαχαρωτό.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.